src/main/kotlin/org/rust/cargo/runconfig/buildtool/RsBuildEventsConverter.kt

```diff
@@ -3,84 +3,69 @@
  * found in the LICENSE file.
  */
 
+@file:Suppress("UnstableApiUsage")
+
 package org.rust.cargo.runconfig.buildtool
 
+import com.google.common.annotations.VisibleForTesting
 import com.google.gson.JsonObject
-import com.google.gson.JsonParser
-import com.google.gson.JsonSyntaxException
 import com.intellij.build.FilePosition
 import com.intellij.build.events.BuildEvent
 import com.intellij.build.events.MessageEvent
 import com.intellij.build.events.StartEvent
 import com.intellij.build.events.impl.*
 import com.intellij.build.output.BuildOutputInstantReader
 import com.intellij.build.output.BuildOutputParser
+import com.intellij.execution.process.AnsiEscapeDecoder
 import com.intellij.openapi.progress.ProgressIndicator
+import com.intellij.webcore.util.JsonUtil.tryParseJsonObject
```

Comment:
I have some doubt it's worth to use something from `com.intellij.webcore` package. If you look where it's located, you will find that it's located in `idea.jar`, i.e. in IDEA-specific library. I'm worried that this module may not be included in some minor IDE. As I can see, we use only `tryParseJsonObject` method from it. Let's better add this method into our codebase to avoid possible errors like `NoSuchMethodError` in some products